homeassistant/components/irm_kmi/__init__.py

```diff
@@ -0,0 +1,39 @@
+"""Integration for IRM KMI weather."""
+
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryError
+
+from .const import PLATFORMS
+from .types import IrmKmiConfigEntry, RuntimeData
```

Comment:
I would also rename it to something more like `IrmKmiData` or something, RuntimeData sounds very generic

Reply:
Renamed in commit 4c4b9e29927df2a6544a3fbd501f09faed607a5b